config/_default/menus/main.en.yaml

```
@@ -3827,96 +3827,101 @@ menu:
       url: tracing/trace_explorer/trace_queries/
       parent: trace_explorer
       weight: 606
+    - name: APM Recommendations
```

```suggestion
    - name: Recommendations
```
What do you think about just putting "Recommendations" as the name in the nav. This is what CCM Recommendations does, and I think it avoids repeating "APM" too many times.